internal/configuration/schema/authentication.go

```diff
@@ -17,7 +17,26 @@ type LDAPAuthenticationBackendConfiguration struct {
 
 // FileAuthenticationBackendConfiguration represents the configuration related to file-based backend
 type FileAuthenticationBackendConfiguration struct {
-	Path string `mapstructure:"path"`
+	Path        string `mapstructure:"path"`
+	Rounds      int    `mapstructure:"rounds"`
```

Comment:
I'd wrap all parameters in a dedicated struct (something like PasswordOptions or equivalent) so that it's clearer for users that those parameters are related to password hashing. It's important since the configuration might be edited by people who don't even know what password hashes are.

Reply:
Done as discussed.